app/celery/provider_tasks.py

```diff
@@ -24,11 +30,18 @@ def deliver_throttled_sms(self, notification_id):
     _deliver_sms(self, notification_id)
 
 
+# Celery rate limits are per worker instance and not a global rate limit.
+# https://docs.celeryproject.org/en/stable/userguide/tasks.html#Task.rate_limit
+# This task is dispatched through the `send-sms-tasks` queue.
+# This queue is consumed by 6 Celery instances with 4 workers in production.
+# The maximum throughput is therefore 6 instances * 4 workers = 24 tasks per second
+# if we set rate_limit="1/s" on the Celery task
 @notify_celery.task(
     bind=True,
     name="deliver_sms",
     max_retries=48,
     default_retry_delay=300,
+    rate_limit="1/s",
 )
```

Comment:
With the rate limit addition, the 2 SMS tasks in this file have the same settings but a different name. We could have a single task, dispatched on different queues. I thought that it was important to limit code changes and get this out quickly in production. A refactor may be needed in a few weeks.

If you think this is a better idea to do it now, I'm happy to make this change.